corfudb-tools/src/main/java/org/corfudb/browser/CorfuBrowser.java

```diff
@@ -0,0 +1,110 @@
+
+    /**
+     * Creates a CorfuBrowser which connects a runtime to the server.
+     * @param host Corfu Server Host
+     * @param port Corfu Server Port
+     * @param keyStore Location of the KeyStore File
+     * @param ksPasswordFile Location of the KeyStore Password File
+     * @param trustStore Location of the Truststore File
+     * @param tsPasswordFile Location of the Truststore Password File
+     * @param tlsEnabled Tls Enabled Status
+     */
+    public CorfuBrowser(String host, int port, String keyStore,
+        String ksPasswordFile, String trustStore, String tsPasswordFile,
+        boolean tlsEnabled) {
+        configureLogger();
+        runtime = CorfuRuntime.fromParameters(
+            CorfuRuntime.CorfuRuntimeParameters.builder().useFastLoader(false)
+        .cacheDisabled(true)
+        .tlsEnabled(tlsEnabled)
+        .keyStore(keyStore)
+        .ksPasswordFile(ksPasswordFile)
+        .trustStore(trustStore)
+        .tsPasswordFile(tsPasswordFile).build());
+        singleNodeEndpoint = String.format("%s:%d", host, port);
+        connectRuntime(singleNodeEndpoint);
+    }
+
+    private void connectRuntime(String endpoint) {
+        runtime.parseConfigurationString(endpoint);
+        runtime.connect();
+        log.info("Successfully connected to {}", endpoint);
+    }
+
+    /**
+     * Setup logback logger
+     * - pick the correct logging level before outputting error messages
+     * - add serverEndpoint information
+     *
+     * @throws JoranException logback exception
+     */
+    private static void configureLogger() {
+        final Logger root = (Logger) LoggerFactory.getLogger(Logger.ROOT_LOGGER_NAME);
+        final Level level = Level.toLevel(LOG_LEVEL);
+        root.setLevel(level);
+    }
+
+    /**
+     * Fetches the table from the given namespace
+     * @param namespace Namespace of the table
+     * @param tableName Tablename
+     * @return CorfuTable
+     */
+    public CorfuTable getTable(String namespace, String tableName) {
+        ISerializer dynamicProtobufSerializer =
+            new DynamicProtobufSerializer(runtime);
+        Serializers.registerSerializer(dynamicProtobufSerializer);
+
+        CorfuTable<CorfuDynamicKey, CorfuDynamicRecord> corfuTable =
+            runtime.getObjectsView().build()
+            .setTypeToken(new TypeToken<CorfuTable<CorfuDynamicKey, CorfuDynamicRecord>>() {
+            })
+            .setStreamName(
+                TableRegistry.getFullyQualifiedTableName(namespace, tableName))
+            .setSerializer(dynamicProtobufSerializer)
+            .open();
+        return corfuTable;
+    }
+
+    /**
+     * Prints the payload and metadata in the given table
+     * @param table
+     */
+    public void printTable(CorfuTable table) {
+        for (Object obj : table.values()) {
+            CorfuDynamicRecord record = (CorfuDynamicRecord)obj;
+            log.info("Payload: {}", record.getPayload());
```

Comment:
It will be nice to print this in table format using  width and precision specifiers
==========
Name: table name  Namespace: namespace
==========
Key   Value
k1      v1
k2      v2

Reply:
I tried this using string.format but to align the keys/values/metadata correctly in their respective columns, there is no certain width which is suitable for all types of data.  Without alignment, it is not readable.

So I am printing it in the form-
Key:
Value:
Meta:
<Row Separator>

Let me know if you want to see a sample output.